### DISCUSSION TITLE
How to make tab bar not transparent/accent colored?

### DISCUSSION THREAD

**Discussion by x24sousa** [black7375/Firefox-UI-Fix#748] [Q&A] [answered]
Hey guys. This UI Fix is exactly what I have been looking for to make Firefox closer to what I remember. 

Firefox: 115
OS: Windows 10

Is there a way to turn off the top tab bar transparency/accent coloration? I'd prefer it to be just a not highlighted version of the regular coloration.

![image](https://github.com/black7375/Firefox-UI-Fix/assets/140837729/a4bbc856-304c-4b6f-bbfe-3f3fcb39df42)

Hopefully that makes sense. 

I do have 

`userChrome.compatibility.accent_color` == false

As well as

`userChrome.compatibility.covered_header_image` == false.

When I have another window take priority, the colors do what I am looking for.

![image](https://github.com/black7375/Firefox-UI-Fix/assets/140837729/05c8d4d2-5965-45c8-b9e8-667e46dac1fd)

Thanks for any help!

**Comment by black7375** [comment, marked as answer]
The background color changes because of the OS settings.
1. settings > Personalization > Colors
2. `Show accent color on title bars and windows borders` to false

Alternatively, if you select a theme other than the default, the accent color will be disabled.

---

You can also choose to remove the code manually.

https://github.com/black7375/Firefox-UI-Fix/blob/7c8a9681524ce409f353b563fe1dc00d21fd8c3a/css/leptonChrome.css#L842-L870

**Reply by x24sousa** [reply]
Thank you for pointing out that code block to me! I commented it out and it's doing exactly what I want now!

I assume I start a new discussion to ask another, slightly different, question?
